src/WordPress/PluginActions.php

```diff
@@ -60,7 +62,6 @@ public function loginWordPress()
      */
     public function getPluginSettings()
     {
-        // Always return true
         $settings = $this->dataStore->getPluginSettings($this->api);
```

Comment:
Don't pass `$this->api` to `CF\WordPress\DataStore.php`.  The DataStore is only for getting/setting data in WordPress, it shouldn't be interacting with the API at all.  It looks like you're using it to create API responses, just do that here instead since you already have access to `$this->api` here.